### ISSUE TITLE
Palworld Crash on Startup After 1.5 Update

### ISSUE THREAD

**Issue by TheNicheGuy** [UE4SS-RE/RE-UE4SS#411]
**Branch or Release**
RE-UE4SS 3.0.1

**Game and Engine Version**
Palworld v0.1.5.0

**Describe the bug**
Immediate crash when I boot up the game, followed by the crash report log window. I was playing on 2/26/2024 from 11AM to 1:45PM with freshly downloaded mods from the same day. When I came home from work around 7:30PM the update had auto downloaded, and when I went to boot the game up it crashes instantly. Tried again with removing two mods but didn't make a change.

**Mods directory**
Mods I added myself: [Mods.zip](https://github.com/UE4SS-RE/RE-UE4SS/files/14425401/Mods.zip)

**To Reproduce**
1. Launch game

**Expected behavior**
The game to boot up and play normally.

**Screenshots, UE4SS Log, and .dmp file**
Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x0000000000000000

Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
kernel32
ntdll

**Desktop (please complete the following information):**
 - OS: [Win11]

**Additional context**
I've added mods to many other games in the past, but I honestly have no idea what I'm talking about when it comes to how mods work, or even code for that matter, but I'm feeling like the update broke how things work for UE4SS and its mods that rely on it. I've removed mods that don't go in that folder, like the ones in "~mods" folder and "Logicmods" folder, but I still experience the issue. I don't use vortex, I download everything manually. I haven't tried a fresh install, yet, but I don't want to unless I have to. Thank you for the help, and I appreciate your time.

**Comment by UE4SS** [comment]
Can you verify whether there's a `.dmp` file in the same directory where you installed UE4SS, and if there is, upload it ? And also upload the log file from the same directory.

**Comment by TheNicheGuy** [comment]
I don't see any .dmp files, you're talking about the Binaries/Win64 Directory folder, right?? Just to make sure I'm looking in the right spot! I'm also getting the feeling that my game files may be messy; I've extracted all possible mods and files that work with them from the folders and my game still will crash. I'm going to run a fresh install to make sure my game files are the issue

**Comment by UE4SS** [comment]
> I don't see any .dmp files, you're talking about the Binaries/Win64 Directory folder, right?? Just to make sure I'm looking in the right spot! I'm also getting the feeling that my game files may be messy; I've extracted all possible mods and files that work with them from the folders and my game still will crash. I'm going to run a fresh install to make sure my game files are the issue

Yes that is the correct directory, assuming the next directory up is `Pal` and not `Engine`.
If the log file (`UE4SS.log`), in the same directory, doesn't exist then it's likely that UE4SS isn't installed properly.

**Comment by TheNicheGuy** [comment]
Ran file integrity after removing everything, runs totally fine now. I added UE4SS back with all mod files I had and it boots normally, but some mods won't activate. I apologize if I've wasted any time here, UE4SS works totally fine it seems it's specific mods themselves that may not be working with the update. I'll reach out to mod authors if I locate specific mods causing any issues, thank you very much for your help and time, I hope you have a wonderful rest of your week!
